helpers/calendar/analytics.py

```diff
@@ -119,3 +120,32 @@ def get_meetings_duration_analytics(self, query, start_date, end_date):  # noqa:
             )
             result.append(output)
         return result
+
+    def get_most_booked_rooms(self, query, start_date, end_date):  # noqa: E501
```

Comment:
Since we are getting booked rooms and using indexing in various resolvers for least booked rooms and most booked rooms, would it be appropriate to just name this function as  get_booked_rooms?